### PR TITLE
Added doc type checker

### PR DESCRIPTION
**Pull request by stobasa**

### CODE REVIEW

File: content_from_pdf.py [hunk 1]
```diff
@@ -1,378 +1,402 @@
...
+                  "Direct co-investing", "Primary issuances", "Secondary purchases", "Minimum", "Soft commitment", "Hard commitment", "Change of control", "Liquidation event", "Liquidity event", "Dividend", "Security", "Gross returns", "Net returns", "Holding period", "MOIC", "MOM", "IRR", "Current yield", "Carried interest", "Promote", "Performance-based fees", "Hurdle rate", "Preferred return", "Waterfall", "Sliding fee scale", "Closing fee", "Management fee", "IOI", "LOI", "SPV", "NDA", "Non-circ", "GP", "LP", "Committed capital", "Fund mandate", "Capital call", "Drawdown", "Vintage year", "J-curve"]
```
Comment (otseobande) [inline content_from_pdf.py]: Breaking this long lists into one keyword per line would make it easier to read.

File: content_from_pdf.py [hunk 2]
```diff
@@ -1,378 +1,402 @@
...
+
+
+def check_doc_type(text_list):
```
Comment (otseobande) [inline content_from_pdf.py]: Please can you move the logic for type checking to a new module (file). This file is already getting too long.

Comment (stobasa) [reply content_from_pdf.py]: Done

File: content_from_pdf.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (otseobande) [inline content_from_pdf.py]: Please remove the git `<<<<<<< HEAD` line here.

File: Doc_checker.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+        return("Hedge Fund")
+    elif percentage_private > percentage_hedge:
+        return("Private Equity")
+
+
+def checking_doc_type_result(filename):
+    text_list = extractText(filename)
+    text_list_refined = str(text_list[0]).replace('\n', ' ')
+    result = check_doc_type(text_list_refined)
+    return result
```
Comment (otseobande) [inline Doc_checker.py]: Good work this file looks clean.

File: content_from_pdf.py (hunk 3 above)
Comment (stobasa) [reply content_from_pdf.py]: Done

**Pull request closed without merging**